scripts/tugboat/init.sh

```diff
@@ -11,4 +11,4 @@ composer install
 $DIR/install-node.sh 4.4.1
 
 
-$DIR/../../blt.sh ci:setup
+$DIR/../.blt ci:setup
```

Comment:
This must remain a file reference. We cannot use an alias in combination with file paths.